react_projects/rick-and-morty-web/src/components/rawItem/RawItem.js

```diff
@@ -0,0 +1,26 @@
+import './RawItem.css'
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+
+
+class RawItem extends Component {
+  render() {
+    let { character } = this.props
```

Comment:
Always use const if you don't change the value. This is good for readability (if you see a `const`, you know that it is not going to be changed).